test-utils/src/main/java/com/criteo/publisher/network/CdbMock.kt

```diff
@@ -347,12 +349,73 @@ class CdbMock(private val jsonSerializer: JsonSerializer) {
     """.trimIndent()
 
       return MockResponse()
-          .setHeader(CONTENT_TYPE, "text/plain; charset=utf-8")
+          .setHeader(CONTENT_TYPE, TEXT_PLAIN)
+          .setBody(response)
+    }
+
+    @Suppress("MaxLineLength")
+    private fun handleVastCasperRequest(request: RecordedRequest): MockResponse {
```

Comment:
I could be wrong but should not this one be `handleVastWrappedCasperRequest` instead of the one below? This one has a **Wrapper** tag in the response and not the other one.

Reply:
This one is wrapping (hence the Wrapper tag) the wrapped VAST. If I rename them to `handleWrappingVastCasperRequest` (this one) and `handleWrappedVastCasperRequest` (the one below), is it clearer then ?

Reply:
Renaming the function or adding a comment is fine by me, it is slightly confusing at first glance because of the Wrapper tag.

Reply:
done